frontend/src/components/settings/SettingInput.tsx

```diff
@@ -153,6 +153,21 @@ const SettingInput: React.FC<RenderSettingInputProps> = ({
             {...rest}
           />
         );
+      } else if (setting.label === "target_block_id") {
+        const { onChange, ...rest } = field;
+
+        return (
+          <AutoCompleteEntitySelect<IBlock, "name", false>
```

Comment:
Let's group the options by flow/category please 🙏

Reply:
Sure :technologist: